Adds a simple command-line interface

gabmontes opened the pull request:
As suggested in #40, this commit adds a `bin` keyword in the `package.json` and a `bin.js` file that takes a title and message from the command line and launches a notification.

Additional configuration is not currently supported but can be easily added through more command line parameters.

Hope this helps.

mikaelbr commented:
Thanks alot. I hadn't gotten around to this yet, so this got me started. I extended your initial work and added more options, and added usage information: 7cd5c2a9fb06ffe17b531d21df9d8f5cdb941a2b :cake:

gabmontes commented:
Great!!!

Can you update the published version at npm too?

Thanks a lot!!!

-----Mensaje original-----
De: "Mikael Brevik" <notifications@github.com>
Enviado el: ‎15/‎04/‎2015 17:20
Para: "mikaelbr/node-notifier" <node-notifier@noreply.github.com>
CC: "Gabriel Montes" <gabriel@utoctadel.com.ar>
Asunto: Re: [node-notifier] Adds a simple command-line interface (#52)

Thanks alot. I hadn't gotten around to this yet, so this got me started. I extended your initial work and added more options, and added usage information: 7cd5c2a  
—
Reply to this email directly or view it on GitHub.

mikaelbr commented:
The latest version is already pushed: https://npmjs.org/package/node-notifier

gabmontes commented:
Thanks again!!!

-----Mensaje original-----
De: "Mikael Brevik" <notifications@github.com>
Enviado el: ‎16/‎04/‎2015 01:54
Para: "mikaelbr/node-notifier" <node-notifier@noreply.github.com>
CC: "Gabriel Montes" <gabriel@utoctadel.com.ar>
Asunto: Re: [node-notifier] Adds a simple command-line interface (#52)

The latest version is already pushed: https://npmjs.org/package/node-notifier
—
Reply to this email directly or view it on GitHub.
